content/pages/meetup/2024/0925-nycu.rst

```diff
@@ -0,0 +1,141 @@
+| 18:30 ~ 19:30   | free chat                                                                                                                       |
++-----------------+--------------------------+-------------------------------------------------------+----------------------------------------------+
+| 19:30 ~ 21:00   | free chat                | modmesh discussion and training :                     | nsd teaching assistance :                    |
+|                 |                          |                                                       |                                              |
+|                 |                          | 1. Use Bezier curve to fit NACA 4-digit airfoil       | 1. project discussion                        |
+|                 |                          | 2. Prototype profiler object serialization using JSON | 2. homework assistance                       |
+|                 |                          | 3. Build modmesh                                      |                                              |
+|                 |                          | 4. Make a simple PR for modmesh                       |                                              |
++-----------------+--------------------------+-------------------------------------------------------+----------------------------------------------+
+| 21:00 ~ 21:30   | free chat                                                                                                                       |                            
++-----------------+--------------------------+-------------------------------------------------------+----------------------------------------------+
+
+
+|
+
+Subjects
+------------------
+
+modmesh discussion and training
++++++++++++++++++++++++++++++++++++++
+
+discussion
+^^^^^^^^^^^^
+
+In the discussion session, 
+we will discuss the following topics simultaneously, 
+and you can choose the topic you are interested in.
+
+1. `Use Bezier curve to fit NACA 4-digit airfoil <https://github.com/solvcon/modmesh/issues/320>`__
+2. `Prototype profiler object serialization using JSON <https://github.com/solvcon/modmesh/issues/343>`__
+
+
+training
+^^^^^^^^^^^^
+
+If you are interested in the `modmesh project <https://github.com/solvcon/modmesh>`__, 
+but you are not familiar with the project, 
+we also have a training session for you, 
+and you can learn part of the following skills.
+
+1. Setting up and demonstrating modmesh.
+2. Basics of Git and GitHub.
+3. Writing Pybind11 and Unittest.
+4. Fundamentals of CMake, GNU Make, and GitHub Workflow.
+
+
+free chat
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+In this session, you could discuss any topic you are interested in. 
+Additionally, we will discuss the following items to prepare the upcoming events.
+
+* Task arrangement for volunteers
+* Promotion plan for upcoming events, including scisprint and next meetup
+* The schedule of the projects
```

Comment:
Here are the suggestions that @meng5113 mentioned.

Reply:
It looks good to me, thanks!

Reply:
> It looks good to me, thanks!

In addition to the inline comment like what you left here, it will help the author to know that you are OK the PR to be merged by adding a global comment saying it explicitly.  You can do that by using the "Review changes" button in the "Files changed" tab.

Reply:
> > It looks good to me, thanks!
> 
> In addition to the inline comment like what you left here, it will help the author to know that you are OK the PR to be merged by adding a global comment saying it explicitly. You can do that by using the "Review changes" button in the "Files changed" tab.

Thanks for the heads up! I've added a global comment.